cvat/apps/engine/tests/_types.py

```diff
@@ -0,0 +1,9 @@
+from typing import Protocol
+
+
+class OrderStrategy(Protocol):
+    def __call__(self, key_path: str) -> bool: ...
+
+
+always_check_order: OrderStrategy = lambda _: True
+never_check_order: OrderStrategy = lambda _: False
```

Comment:
These 2 modes could probably be represented simply by the `True` and `False` values

Reply:
I mean, why not define `check_order: bool | OrderStrategy = True,` instead?